exercises/branching-conditionals/rockPaperScissorsWinner/rockPaperScissorsWinner.js

```diff
@@ -24,15 +24,49 @@
  * @returns {string} Returns 'win', 'lose', or 'draw' depending on
  *  whether the first player won, lost, or drew the match.
  */
-function rockPaperScissorsWinner(array) {
+function rockPaperScissorsWinner(firstPlayerMove, secondPlayerMove) {
   // This is your job. :)
+  if ((!Number.isInteger(firstPlayerMove) || (firstPlayerMove > 2 || firstPlayerMove < 0)) || (!Number.isInteger(secondPlayerMove) || (secondPlayerMove > 2 || secondPlayerMove < 0))){
```

Comment:
To handle errors like this, you probably want to break them up into cases in order to improve the error messages and better direct the debugging efforts of anyone who reads them, e.g.,

```js
if (!Number.isInteger(firstPlayerMove)) {
  throw new Error(...);
}

if (!Number.isInteger(secondPlayerMove)) {
  throw new Error(...);
}

if (firstPlayerMove < 1 || firstPlayerMove > 2) {
  throw new Error(...);
}

if (secondPlayerMove < 1 || secondPlayerMove > 2) {
  throw new Error(...);
}
```

Each `new Error(...)` would have a message specific to the particular condition.